doc/main.html

```diff
@@ -610,15 +610,21 @@ <h4>Additional elements section</h4>
         <ul>
           <li>an <code>id</code> attribute whose value is prefixed with <code>element-</code>;</li>
           <li>a <code>title</code> attribute that provides a short description of the element;</li>
-          <li>a <code>href</code> attribute that links to the element itself, typically in the <code>doc/elements</code>
-          directory.</li>
+          <li>a <code>href</code> attribute that links to the element itself</li>
```

Comment:
```suggestion
          <li>a <code>href</code> attribute that links to the element, as detailed below.</li>
```

Reply:
updated in https://github.com/SMPTE/html-pub/pull/137/commits/b72678433517467034a0ee22a61d202f3d65e3a3